lsp-perl.el

```diff
@@ -50,6 +50,29 @@
   :group 'lsp-perl
   :package-version '(lsp-mode . "6.3"))
 
+(defcustom lsp-perl-perlCmd nil
+  "Path to perl interpreter used in Perl Language Server.  Defaults to `perl' if nil."
+  :type 'string
+  :group 'lsp-perl)
+(defcustom lsp-perl-perlInc nil
```

Comment:
nit: can you change the name so they're all using dash words instead of Camel case?